filter.py

```diff
@@ -1,51 +1,54 @@
-#--------Helper Functions to be used for sorting and filtering-------#
+"""Helper Functions to be used for sorting and filtering."""
 import datetime
 
 
-# Filters by all of the criteria shown below, returns an array of filtered releases
-def filter(data, state, label, start_date, end_date, datetype):
+def filter_releases(data, state, label, start_date, end_date, datetype):
+  """Filters by all of the criteria shown below, returns an array of filtered releases."""  # pylint: disable=line-too-long
+  # convert variables from unix type to something usable
+  state = int(state)
```

Comment:
why is state an int ? Where is the release Object defined ?

Reply:
State is an int because in the original .proto file state was of type Status, and status had 4 different possible options so we thought we could associate each status type with an int. In the UI these show up as different symbols rather than the word itself. 

Also, right now there's no release Object, rather each release is a dictionary. You can see the structure of the dictionary in the json files fake_release_data and fake_task_data. Would you want us to create objects for both of these?

Reply:
You can use a string enum. That way you won't have to hard code it in the javascript either.

Yes please define release and task objects. The JSON parsing should return directly objects not map.